drivers/display/ft8xx/ft8xx.c

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright (c) 2020 Hubert Miś
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <drivers/display/ft8xx/ft8xx.h>
```

Comment:
It looks like this is a local private header, so you don't need to use full path, just use `"ft8xx.h"`

Reply:
It's actually a public header placed in `include/drivers/display/ft8xx`. Should I remove some directories from the path? I thought that it is better to make include paths explicit.